### ISSUE TITLE
[Initials] Add Parentheses and Brackets

### ISSUE THREAD

**Issue by amarakon** [georgd/EB-Garamond#175]
I normally think that sentences should not begin with parentheses or brackets, but I am typesetting an English translation of the Qur’ān with each chapter beginning with an initial capital with the EB Garamond Initials font, and some of the chapters begin with parentheses. Just like how EB Garamond Initials has support for guillemets and quotation marks, I think it should also support parentheses, brackets, and top half brackets (`U+2E22` and `U+2E23` respectively).

**Comment by georgd** [comment]
Sounds legitimate to me. Out of curiosity, when you say chapter, you mean the surah? What kind of contents are there in parentheses?

**Comment by amarakon** [comment]
> Out of curiosity, when you say chapter, you mean the surah?

Yes, I mean the surah, but I used the word *chapter* because it is more familiar to most people. In the actual book, I use the word *surah*.

> What kind of contents are there in parentheses?

All translated words that are not part of the original text but that are necessary to understand it because that is what the translator believes is implied. This is actually [what top half brackets are used for](https://en.wikipedia.org/wiki/Bracket#Quine_corners_%E2%8C%9C%E2%8C%9D_and_half_brackets_%E2%B8%A4_%E2%B8%A5_or_%E2%B8%A2_%E2%B8%A3), but parentheses are used in this translation instead of them. For this reason, I plan to use a global string substitution to replace all parentheses with top half brackets. I think this is legal for me to do without permission since I am not publishing it, but please correct me if I am wrong.

Here is a list of surahs/chapters that start with an opening parenthesis in the Abdullah Yusuf Ali translation:

16. An-Nahl (The Bee)
55. Ar-Rahman (The Beneficent)
80. Abasa (He Frowned)

**Comment by georgd** [comment]
Thanks a lot for the explanation.
